benchbuild/experiment.py

```diff
@@ -112,7 +119,7 @@
         return _id
 
     @id.validator
-    def validate_id(self, _, new_id):
+    def validate_id(self, _: tp.Any, new_id: uuid.UUID) -> None:
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)